### PR TITLE
Add aws vpc cni metrics helper

### LINKED ISSUE #155: Metrics Helper for AWS VPC CNI
The CNI Metrics helper resources [found here](https://github.com/aws/amazon-vpc-cni-k8s/tree/master/config/v1.6) need to be reflected in the [aws-vpc-cni chart](../tree/master/stable/aws-vpc-cni). If given the green light I can gladly add this over the next few days

### PR DESCRIPTION
**Pull request by TechDawg270**
Issue #, if available:
closes #155 

Description of changes:
* Added conditional creation of [cni-metrics-helper](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/config/v1.6/cni-metrics-helper.yaml)

* Bumped image and appVersion to latest of `v1.6.2`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: stable/aws-vpc-cni/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ nameOverride: aws-node
 
 image:
   region: us-west-2
-  tag: v1.6.1
+  tag: v1.6.2
```
Comment (max-rocket-internet) [inline stable/aws-vpc-cni/values.yaml:11]: You're not just adding `cni-metrics-helper` but you're also bumping the version. Are you sure there's no other required changes for `v1.6.2`? Sometimes other parts of the chart need to change with new CNI releases, e.g. https://github.com/aws/eks-charts/pull/74

Comment (max-rocket-internet) [reply stable/aws-vpc-cni/values.yaml:11]: Never mind, I checked, there's no other changes required: https://github.com/aws/amazon-vpc-cni-k8s/commits/master/config/v1.6/aws-k8s-cni.yaml

File: stable/aws-vpc-cni/templates/cni-metrics-helper.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    verbs: ["list", "watch"]
+  - apiGroups: ["batch"]
+    resources:
+      - cronjobs
+      - jobs
+    verbs: ["list", "watch"]
+  - apiGroups: ["autoscaling"]
+    resources:
+      - horizontalpodautoscalers
+    verbs: ["list", "watch"]
```
Comment (jaypipes) [inline stable/aws-vpc-cni/templates/cni-metrics-helper.yaml:42]: The above seems like a very broad ruleset to give to the service account for the metrics helper. Are we sure the above is necessary?

Comment (TechDawg270) [reply stable/aws-vpc-cni/templates/cni-metrics-helper.yaml:42]: it was taken directly from [the upstream yaml manifest](https://github.com/aws/amazon-vpc-cni-k8s/blob/master/config/v1.6/cni-metrics-helper.yaml)

**Review by jayanthvn — CHANGES_REQUESTED** [review]
Current EKS default is v1.7.5. Can you please update it?

**Comment by jdn5126** [comment]
Closing as `cni-metrics-helper` is already vendored as a separate chart.

**Pull request closed without merging**